docs/.vuepress/plugins/chart-editor/components/ChartEditor.vue

```diff
@@ -13,15 +13,37 @@
 </template>
 
 <script>
-import {Chart} from 'chart.js';
-import * as Utils from '../utils';
+import {
+  ArcElement, BarController, BarElement,
+  BubbleController, CategoryScale,
+  Chart, DoughnutController, Filler, Legend, LinearScale, LineController, LineElement,
+  PieController, PointElement,
+  PolarAreaController, RadarController, RadialLinearScale,
+  ScatterController,
+  TimeSeriesScale,
+  Title, Tooltip
+} from 'chart.js';
```

Comment:
I created https://github.com/chartjs/Chart.js/pull/8419 as a potential solution to this problem. What do you think?

Reply:
In my migration to V3 I used this code to make sure I had everything imported, dont know if its easyer to read but its less lines:

```
import * as ChartJs from 'chart.js';

ChartJs.Chart.register.apply(null, Object.values(ChartJs).filter((chartClass) => (chartClass.id)));
```

Reply:
Thanks @etimberg!

@LeeLenaleee It's definitely shorter but a bit hacky. It's also not very robust since it will try to register anything with an `id`, which could be something not registrable. I don't think we should promote this method to Chart.js users.

Reply:
Thanks, will address once https://github.com/chartjs/Chart.js/pull/8425/files is released (beta.11?)